### PR TITLE
fix(sharing): Move video verification option to password and hide without Talk

### LINKED ISSUE #41244: [Bug]: When customizing a link share there is the option "Video verification" despite the "Nextcloud Talk" App isn't installed on the instance
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on [Github](https://github.com/nextcloud/server/issues?q=is%3Aopen+is%3Aissue+label%3Abug) OR [Nextcloud Community Forum](https://help.nextcloud.com/) _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

When customizing a link share there is the option "Video verification" despite the "Nextcloud Talk" App isn't installed on the instance.
By clicking the option there are appearing two error notifications.
```
"Sending the password by Nextcloud Talk" for sharing a file or folder failed because Nextcloud Talk is not enabled.
Error updating the share: "Sending the password by Nextcloud Talk" for sharing a file or folder failed because Nextcloud Talk is not enabled.
```
I tested it on versions 27.1.2 and 27.1.3

### Steps to reproduce

1. Click on "share"
2. Create new share link
3. Open "customize link"
4. Click on "Video verification"

### Expected behavior

If the Nextcloud Talk App isn't installed on an instance, this option shouldn't be available like in the versions before.
Or there should be an options in "Administration Settings" -> "Sharing" to disable this feature completely.

### Installation method

None

### Nextcloud Server version

27

### Operating system

None

### PHP engine version

None

### Web server

None

### Database engine version

None

### Is this bug present after an update or on a fresh install?

None

### Are you using the Nextcloud Server Encryption module?

None

### What user-backends are you using?

- [ ] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

_No response_

### List of activated Apps

_No response_

### Nextcloud Signing status

_No response_

### Nextcloud Logs

_No response_

### Additional info

_No response_

### PR DESCRIPTION
**Pull request by nickvergessen**
* Resolves: #41244 

## TODO

- [x] Move it right below the password option which makes it visible
- [x] Hide when Talk is not enabled

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [x] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

File: apps/files_sharing/src/views/SharingDetailsTab.vue [hunk 1]
```diff
@@ -618,8 +618,8 @@ export default {
 				return false
 			}
 
-			// Anything else should be fine
-			return true
+			// Is Talk enabled?
+			return OC.appswebroots.spreed !== undefined
```
Comment (nickvergessen) [inline apps/files_sharing/src/views/SharingDetailsTab.vue:622]: Same logic as 26:
https://github.com/nextcloud/server/blob/ef4a861b5cf98a5b6e90e346e98ed2dad4ce9d98/apps/files_sharing/src/components/SharingEntryLink.vue#L453-L462

**Review by Antreesy — APPROVED** [review]
Tested with Talk app enabled/disabled

**Review by AndyScherzinger — APPROVED** [review]

**Pull request merged**